MapboxNavigation/RouteMapViewController.swift

```diff
@@ -541,6 +541,10 @@ extension RouteMapViewController: RoutePageViewControllerDelegate {
                 mapView.setUserTrackingMode(.followWithCourse, animated: true)
             }
         }
+        
+        if let stepIndex = routeController.routeProgress.currentLeg.steps.index(where: { $0 == step }) {
+            mapView.addArrow(route: routeController.routeProgress.route, legIndex: routeController.routeProgress.legIndex, stepIndex: stepIndex)
```

Comment:
We'll have to figure out how to handle multiple legs here. I'm not sure even what happens when you try to swipe past the current leg.